package.json

```diff
@@ -116,6 +116,7 @@
     "react-dnd": "^2.4.0",
     "react-dnd-touch-backend": "0.2.7",
     "react-highcharts": "^15.0.0",
+    "react-is": "16.8.2",
```

Comment:
is locking this down to a patch version a deliberate choice?

Reply:
The `react` dependency is locked so this one needs to be for now